packages/pluggableWidgets/color-picker-web/src/ColorPicker.xml

```diff
@@ -1,76 +1,69 @@
->
+<widget id="com.mendix.widget.custom.colorpicker.ColorPicker" needsEntityContext="true" pluginWidget="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../../node_modules/mendix/custom_widget.xsd">
     <name>Color picker</name>
     <description>Pick a color from color input</description>
     <studioProCategory>Display</studioProCategory>
     <studioCategory>Display</studioCategory>
     <properties>
-        <propertyGroup caption="General">
-            <propertyGroup caption="Data source">
-                <property key="colorAttribute" type="attribute">
-                    <caption>Color attribute</caption>
-                    <description>Attribute containing a valid color like #d0d0d0, rgb(115,159,159) or rgba(195,226,226,1). Non-code color values such as 'red' are not supported
-                    </description>
-                    <attributeTypes>
-                        <attributeType name="String" />
-                    </attributeTypes>
-                </property>
-                <property key="format" type="enumeration" defaultValue="hex">
-                    <caption>Color format</caption>
-                    <description>The output formatting of the selected color</description>
-                    <enumerationValues>
-                        <enumerationValue key="hex">HEX</enumerationValue>
-                        <enumerationValue key="rgb">RGB</enumerationValue>
-                        <enumerationValue key="rgba">RBGA</enumerationValue>
-                    </enumerationValues>
-                </property>
-            </propertyGroup>
-            <propertyGroup caption="Label">
-                <systemProperty key="Label" />
-            </propertyGroup>
-            <propertyGroup caption="Editability">
-                <systemProperty key="Editability" />
-            </propertyGroup>
+        <propertyGroup caption="Data source">
+            <property key="colorAttribute" type="attribute">
+                <caption>Color attribute</caption>
+                <description>The attribute containing a valid color, supported color formats are hexadecimal, rgb and rgba. Non-color formats such as ‘red’ are not supported.</description>
+                <attributeTypes>
+                    <attributeType name="String" />
+                </attributeTypes>
+            </property>
         </propertyGroup>
-        <propertyGroup caption="Appearance">
-            <property key="mode" type="enumeration" defaultValue="input">
-                <caption>Mode</caption>
+        <propertyGroup caption="General">
+            <property key="advanced" type="boolean" defaultValue="false">
+                <caption>Enable advanced options</caption>
                 <description />
+            </property>
+            <property key="displayMode" type="enumeration" defaultValue="popover">
```

Comment:
This will cause a breaking change, the keys should remain the same as before, just the captions should be changed